_posts/2022-10-03-the-definitive-guide-2.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Refresh of Trino: The Definitive Guide"
+author: Manfred Moser, Martin Traverso, Matt Fuller
+excerpt_separator: <!--more-->
+---
+
+It was time for a refresh. Only a little while ago, in April 2021, we announced
+the [Trino version of our definitive guide]({{ site.url }}{% post_url
+2021-04-21-the-definitive-guide %}). But again Trino as a project and community
+has continued to innovate and grow. Numerous smaller and larger details changed,
+and the examples and resources really needed to be fixed.
+
+Today, we are happy to announce that after a few months of updates, testing, and
+editing the second edition of __Trino: The Definitive Guide__ is available.
+
+> ## [Get a free from Starburst now!](https://www.starburst.io/info/oreilly-trino-guide/)
+
+<!--more-->
+
+<img src="/assets/ttdg2-cover.png" align="right" style="float: right; margin-left: 20px; margin-bottom: 20px; width: 100%; max-width: 350px;">
+
+The new edition of the book from O'Reilly is available in digital formats
+as well as physical copies. You can find more information about the book on [our
+permanent page about it](/trino-the-definitive-guide.html).
+
+The book is now updated to Trino release 392 for all filenames, installation
+methods, commands, names and properties. We addressed all problems found by our
+readers and reported to us as well.
+
+We updated to Java 17 usage, added more SQL statements, and added info about
+[Python tools like dbt]]({{ site.url }}{% post_url 2022-09-20-python-progress
+%}) and clients like Metabase. We talk about the lakehouse architecture, and new
+connectors like Iceberg and Delta Lake.
+
+So what are you waiting for? Go get a copy, check out the [updated example code
+repository]({{site.github_org_url}}/trino-the-definitive-guide),
+provide feedback and contact us on [Slack](/slack.html).
+
+*Manfred, Martin, and Matt*
```

Comment:
Don't forget to mention Trino Summit and link to the other blog post. We should make the autographed copies a big deal. Maybe we should get your pictures up there.

Reply:
Ok ..